procedures/accounts_setup.md

```diff
@@ -0,0 +1,250 @@
+```
+$ gcloud source repos clone releng-secrets-global --project=moz-fx-releng-secrets-global
+$ gcloud source repos clone secrets-sops-relengworker --project=moz-fx-relengworker-prod-a67d
+```
+#### install sops
+```
+$ brew install sops
+```
+
+#### acquire new user credentials to talk to the Google Cloud API
+```
+gcloud auth application-default login
+```
+
+#### celebrate by operating the sops credentials
+Have a look in the COOKBOOK in the global SOPS repo for more instructions on how to read encrypt/decrypt the files.
+
+:warning:
+    Talk to your manager to add your fingerprint in the SOPS repo and also grant you access in both repos server-side.
+
+### Disclaimer: Changes ongoing after August 2021
+The max session for SOPS credentials has been reset to 24h so there's some commands
+that need rerunning in the shell in order to unblock using SOPS.
+
+```
+$ gcloud auth login
+$ gcloud auth application-default login
+```
+
+### Sharing secrets with a co-worker
+On a side-note, for sharing secrets between employees, [read these docs](https://mana.mozilla.org/wiki/display/SVCOPS/Sharing+a+secret+with+a+coworker).
+
+## Communication:
+
+### Mail
+Mozilla mail is handled by Gmail now.
+
+#### Mail filtering
+
+With all that new email, you will want to set up some filters in Gmail (https://mail.google.com/mail/u/0/#settings/filters) to filter some of the higher-volume automated mail into a folder.
+You may eventually want to handle this information, but on day one hundreds of automation notifications are not going to be educational.
+
+:warning:
+    Please ask your manager to provide an imperfect but useful set of already existing filters to help handling the load.
+
+#### Mailing lists
+
+* You should be added to the release@mozilla.com google group as a new hire. This mailing list is managed by Google groups. Owners of this group will be able to add you. Send a test message to release@m.c to verify that your address has been added/subscribed. Talk to your manager if it is not working.
+* You should be added to the releng@mozilla.com google group as a new hire. This mailing list is managed by Google groups. Owners of this group will be able to add you. Send a test message to releng@m.c to verify that your address has been added/subscribed. Talk to your manager if it is not working.
+
+:warning:
+    release@m.c can contain security-sensitive information. Do not automatically forward your email to a system that is not under Mozilla's control. Same goes for releng@mozilla.com.
+
+### Calendar
+
+Like mail, we now use [Google calendar](https://www.google.com/calendar/)
+
+You'll want to subscribe to the following public calendars:
+* [RelEng MoCo calendar](https://calendar.google.com/calendar/u/0?cid=bW96aWxsYS5jb21fMmQzMjM0MzMzMzM1MzAzNjMxMmQzOTM3MzdAcmVzb3VyY2UuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+* [RelEng PTO calendar](https://calendar.google.com/calendar/u/0?cid=Y190cWEzaHQ5bXB0ODN1djJzZDJsYjVsaTVva0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
```

Comment:
Worth mentioning "Public - Release Engineering" and "Releases Scheduling"?

Reply:
Huh, we no longer use the first one. Or it's been renamed to `RelEng MoCo calendar`. Can you give me a link?

Reply:
`Release Scheduling` is mentioned as part of Releasduty training but I believe it's a good point to circuit it here. Better duplication when it comes to this. Done.

Reply:
If there's one and I'm wrong, I'll follow-up with another PR, resolving for now.